experiment/references.md

```diff
@@ -1,2 +1,3 @@
-1.  [Recursion](http://www.exforsys.com/tutorials/c-language/c-pointers.html)
-2.  [Recursion - Pointer](http://en.wikipedia.org/wiki/Pointer_%28computer_science%29)
+1. [Recursion](http://www.exforsys.com/tutorials/c-language/c-pointers.html)
```

Comment:
Why do we have references to articles on pointers under recursion?
Also, never use http links

Reply:
These already existed in the old references page, so didn't change thinking that they may be relevant or taken reference from.

Reply:
Let me know if they should be removed or left as it is.

Reply:
They stay only if they are relevant. Also strictly no http links, all must be https

Reply:
Removed the two old links.